Develop

Barosz30 opened the pull request:
- [DEMO LINK](https://Barosz30.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Barosz30.github.io/layout_calendar/report/html_report/)

You can modify calendar length and starting day as variables in variables.scss

mgrinko committed:
Update checklist.md

Barosz30 committed:
calendar first try

Barosz30 committed:
fixed readme

choeqq commented on src/styles/blocks/_calendar.scss:
```diff
@@ -0,0 +1,62 @@
+  .calendar__day:nth-child(#{$number}) {
+
+    &::before {
+      content: "#{$day} \a #{$number}";
```
actually the calender day cell should only contain a number of the day

Barosz30 replied:
I know. Just wanted to check how to do it, to improve myself ;) Thanks for checking it out.

choeqq commented on src/styles/blocks/_calendar.scss:
```diff
@@ -0,0 +1,62 @@
+  $numbers: map-merge($numbers, ($i: $day-name));
+}
+
+.calendar {
```
please make sure your calendar is centered, both horizontally and vertically

choeqq reviewed (changes requested):
Hi, my name is Kacper, I am a mentor in other group, and I'll be helping mentors team with checking homework, until they'll catch up with your group's work 😄 

Please make sure you can modify the calendar length and starting day of the calendar (accroding to readme) and you'll be good to go ✅

Barosz30 committed:
implemented tips

choeqq commented on src/index.html:
```diff
@@ -7,6 +7,38 @@
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length">
+      <div class="calendar__day calendar__day--first"></div>
```
```suggestion
    <div class="calendar calendar--month-length calendar__day--first">
      <div class="calendar__day"></div>
```
here I should be able to pass certain month length: 28, 29, 30, 31 and certain starting day: mon, tue, wed ...
you should be able to modify them in devtools, according to README, and your calendar should behave correctly

Barosz30 committed:
lenght and daystart in html

Barosz30 committed:
general fixes

mvjl000 commented on src/styles/blocks/_calendar.scss:
```diff
@@ -0,0 +1,67 @@
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 720px;
```
You can't set width that way.
It needs to be calculated basing on number of days, calendar day size and gaps, paddings

Barosz30 replied:
width: 7 * 100px + 6 * 1px + 2 * 10px;

You mean calculation like this, or saving them all in variables, and then putting them into code?

Barosz30 committed:
added variables for width calculation

pawel-peksa reviewed (approved):
Good job 🔥

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #5 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
